.github/workflows/install.yml

```
@@ -0,0 +1,31 @@
+name: Install to Ubuntu
+on:
+  schedule:
+    - cron: |
+        0 0 * * *
+jobs:
+  install:
+    name: Install to Ubuntu
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-18.04
+            package: mysql-server
+          - os: ubuntu-20.04
+            package: mariadb-server
+          - os: ubuntu-20.10
+            package: mariadb-server
```

We'll change package name by the next release:

```suggestion
            package: mariadb
```